tests/test_smoke.py

```diff
@@ -2380,25 +2383,64 @@ def tmp_copy_mnt_existing_storage_obj(self, tmp_scratch_storage_obj):
     @pytest.fixture
     def tmp_awscli_bucket(self, tmp_bucket_name):
         # Creates a temporary bucket using awscli
-        subprocess.check_call(['aws', 's3', 'mb', f's3://{tmp_bucket_name}'])
+        #subprocess.check_call(['aws', 's3', 'mb', f's3://{tmp_bucket_name}'])
+        try:
+            subprocess.check_output(
+                ['aws', 's3', 'mb', f's3://{tmp_bucket_name}'],
+                stderr=subprocess.PIPE)
+        except subprocess.CalledProcessError as e:
+            # To avoid errors in test_storage_refresh due to immediate
+            # s3 bucket recreation after deletion, unique name is created.
+            stderr = e.stderr.decode()
+            if 'OperationAborted' in stderr:
+                tmp_bucket_name = f'{tmp_bucket_name}-2'
+                subprocess.check_call(
+                    ['aws', 's3', 'mb', f's3://{tmp_bucket_name}'])
+        cmd = f'aws s3api put-bucket-tagging --bucket {tmp_bucket_name} --tagging \'TagSet=[{{Key=skymanaged,Value=sky}}]\''
+        subprocess.check_call(cmd, shell=True)
```

Comment:
do we want this added for all tests that depend on `tmp_awscli_bucket` (i.e., `test_upload_to_existing_bucket`)?

Reply:
I guess it seems more safe to limit the tagging to `test_storage_refresh` only.